src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2505,7 +2505,8 @@ public static string LastWriteTimeString(PSObject instance)
 
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Validate Update-Help from the Web for all PowerShell modules..Validate Update-Help for module 'Microsoft.PowerShell.Core' with scope as 'False'
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /Users/runner/runners/2.164.6/work/1/s/test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1: line 181
181:     $helpFilesInstalled.Count | Should -Be $expectedHelpFiles.Count
```